scss/_patterns_equal-height-row.scss

```diff
@@ -5,6 +5,17 @@
     @extend %vf-row;
     position: relative;
 
+    // Required to override styles in %vf-row
+    @supports (display: grid) {
```

Comment:
Why is this needed? Isn't it exactly what `%vf-row` does already?

https://github.com/canonical/vanilla-framework/blob/4c15e4009ba37baa3c6b7ef4a6b3f386d766a461/scss/_base_grid-definitions.scss#L37-L38